extension/chrome/elements/composer/composer-pwd-or-pubkey-container.ts

```diff
@@ -33,7 +54,7 @@ export class ComposerPwdOrPubkeyContainer extends ComposerComponent {
       this.hideMsgPwdUi(); // Hide 'Add Pasword' prompt if there are no recipients or message is not encrypted
       this.composer.sendBtn.enableBtn();
     } else if (this.composer.recipients.getRecipients().find(r => r.status === RecipientStatuses.NO_PGP)) {
-      this.showMsgPwdUiAndColorBtn();
+      this.showMsgPwdUiAndColorBtn(); // tslint:disable-line:no-floating-promises
```

Comment:
@tomholub is it okay to leave this promise floating? If I handle it with `await`, then I have to turn `showHideContainerAndColorSendBtn` into `async`, and that will cause a whole bunch of changes from `()` to `async ()` in the codebase.

Reply:
Sorry missed your question, answered in https://github.com/FlowCrypt/flowcrypt-browser/pull/2495#discussion_r370803385